app/views/scheduler/index.html.haml

```diff
@@ -9,8 +9,11 @@
           = text_field_tag :course_title, nil, placeholder: 'Add course instance', class: 'form-control'
 
         = form_for :semester, url: scheduler_path, method: :get, remote: true, html: {id: :semester_form} do |f|
-          = f.select :semester, %w(Fall Spring Summer).map { |str| [str, str.downcase] }
-          = f.select :year, options_for_select(2008..DateTime.now.year, DateTime.now.year)
+          = f.select :semester, options_for_select( |
+            %w(Fall Spring Summer).map {|str| [str, str.downcase]}, |
```

Comment:
To your first point: the pipe on the second line immediately follows a comma, so it seems like we should be able to just remove the pipe there.

To your second point (about `downcase`): if you just use `%w(fall spring summer)`, then I believe the text will show up downcased in the browser. Which means I have to ask: have you been verifying your changes locally in the browser?